app/uk/gov/hmrc/transitmovementsrouter/models/errors/ObjectStoreError.scala

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.transitmovementsrouter.models.errors
```

Comment:
This error class need to be moved to service package. uk.gov.hmrc.transitmovementsrouter.services.error

Reply:
No, this is in the right place, it's where all our other errors are.

Reply:
But if you can see RoutingError that is defined inside package uk.gov.hmrc.transitmovementsrouter.services.error. what is the purpose of creating error package inside service then?

Reply:
That too is in the wrong place.

Reply:
That should be move to model then. but not using this ticket.

Reply:
Indeed.